packages/core/src/lib/database/migrations/1743449844752-CreateIntegrationZapierWebhookRepositoryTable.ts

```
@@ -0,0 +1,209 @@
+
+export class CreateIntegrationZapierWebhookRepositoryTable1743449844752 implements MigrationInterface {
+
+    name = 'CreateIntegrationZapierWebhookRepositoryTable1743449844752';
+
+    /**
+     * Up Migration
+     *
+     * @param queryRunner
+     */
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        console.log(chalk.yellow(this.name + ' start running!'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)